packages/cli/src/commands/env/add.ts

```diff
@@ -40,7 +46,7 @@ export default class EnvAdd extends AuthCommand {
       envValue = await ux.prompt(`What is the value of ${envVariableName}?`, { type: 'mask' })
     }
     try {
-      await api.environmentVariables.add(envVariableName, envValue, locked)
+      await api.environmentVariables.add(envVariableName, envValue, locked, secret)
       this.log(`Environment variable ${envVariableName} added.`)
```

Comment:
Do we want to tweak this a bit to say something like `Secret environment variable added` in case the `s` flag is used?

Reply:
Updated